### DISCUSSION TITLE
How to make server accessible outside local network?

### DISCUSSION THREAD

**Discussion by EmilMemil** [zorchenhimer/MovieNight#203] [Q&A]
As the title suggests ive been having trouble making the server accessible from outside my local network, ive made firewall rules inbound for 8089 TCP/UDP, sadly my office router cannot be accessed, apart from that im not sure what i can, any suggestions? thanks

**Comment by zorglube** [comment]
Have a look here, maybe you'll find something https://github.com/zorchenhimer/MovieNight/issues/149#issuecomment-760810280
